docs/Custom_Operation_for_GPUs.md

```diff
@@ -1123,7 +1124,7 @@ const T *UnpackDescriptor(const char *opaque, std::size_t opaque_len) {
 #endif
 ```
 
-#### `gpu_ops/kernels.h`
+### `gpu_ops/kernels.h`
```

Comment:
I think I may have commented this on the other PR by mistake: I think these should remain fourth-level headings so that they fall within the ```### `gpu_ops` code listing``` section.

Reply:
Applied to four headings. Thanks @jakevdp